src/serac/physics/tests/legacy_thermal_expansion.cpp

```diff
@@ -71,9 +71,6 @@ TEST(SolidLegacy, ThermalExpansion)
 
   solid_solver.setThermalExpansion(std::move(cte), std::move(ref_temp), temp);
```

Comment:
Should this file exist at this point? It is not being actually added as a test... or should I add it to the CMake file?

Reply:
This was probably a miss at some point. As we're about to delete capability anyway, I say delete it.